python/cudf_polars/pyproject.toml

```diff
@@ -58,6 +58,8 @@ exclude_also = [
   "class .*\\bProtocol\\):",
   "assert_never\\("
 ]
+# When running the test suite we don't use the plugin
```

Comment:
This comment is confusing. The important point here is that we don't care to measure coverage of the plugin, right? But also, why wouldn't we use the plugin?

Reply:
Updated comment, we don't use the plugin when running the cudf-polars test suite (which is when we're measuring coverage)